soccer/gameplay/robocup-py.cpp

```diff
@@ -326,6 +500,26 @@ BOOST_PYTHON_MODULE(robocup)
 	class_<Geometry2d::Circle, bases<Geometry2d::Shape> >("Circle", init<Geometry2d::Point, float>())
 		.def("intersects_line", &Circle_intersects_line)
 		.def("nearest_point", &Geometry2d::Circle::nearestPoint)
+        .def("contains_point", &Geometry2d::Circle::containsPoint)
+        .def("center", &Circle_get_center)
```

Comment:
Yeah honestly GitHub's review system is kind of garbage.  I with there was a checkbox for "resolved" on the line notes and an easy way to view all unresolved ones.